code/game/objects/items/stacks/stack.dm

```diff
@@ -12,21 +12,33 @@
 /obj/item/stack
 	icon = 'icons/obj/stack_objects.dmi'
 	gender = PLURAL
+	///The list recipes you can make with the stack
 	var/list/datum/stack_recipe/recipes
```

Comment:
these docs are fine but all of these changes are completely unrelated and don't follow good atomic PR standards. At least add to CL I guess